packages/graph/Cargo.toml

```
@@ -0,0 +1,22 @@
+[package]
+name = "graph"
+version = "0.0.0"
+edition = "2021"
+authors = ["HASH"]
+license-file = "LICENSE.md"
+description = "Datastore infrastructure for HASH"
```

```suggestion
description = "The entity-graph query-layer for the HASH datastore"
```

What do you reckon @thehabbos007 @benwerner01?

I'm expecting this Rust crate to be only about adding _graph_ logic on top of the datastore, but I'm not sure all of the infrastructure and such will be managed by the crate so it depends on what we're describing in this `Cargo.toml` (e.g. I don't think we'll be including terraform files or whatever under the scope of this description)

---

That was also my instinct

---

I think that suggestion is good. Infrastructure could be confused with deployment infra

---

Good suggestion, applied it in b5a4d479d17b226fb8d6c5d6289688a422920248